Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -1,17 +1,25 @@
 # -*- coding: utf-8 -*-
-
 import weakref # Using this to prevent problems with garbage collection
 
 from core.jsr223 import scope
 from core.log import logging, LOG_PREFIX
+
+import sys ############ TEMP TODO:
 from core.utils import isActive, getItemValue, postUpdateCheckFirst, sendCommandCheckFirst, kw
-from community.idealarm import custom
+reload(sys.modules['core.utils'])
+from core.utils import isActive, getItemValue, postUpdateCheckFirst, sendCommandCheckFirst, kw
+
+import sys # Trying to reload custom from personal.idealarm TODO: Does this actually work? 
+from personal.idealarm import custom
+reload(sys.modules['personal.idealarm'])
+from personal.idealarm import custom
 
 from org.eclipse.smarthome.model.persistence.extensions import PersistenceExtensions
 
 from org.joda.time import DateTime
 
 log = logging.getLogger(LOG_PREFIX + '.ideAlarm')
+from logging import DEBUG, INFO, WARNING, ERROR
```

Comment:
Logging is in Python though, and Jython is probably never going to go beyond Python 2.7. So, I understand your point, but I don't see it as a concern. 

Now, we're probably going to be using GraalVM eventually, so will be using the latest Python. But setLevel still takes a string for the logging level in Python 3.7.

Reply:
The levels  `DEBUG`, `INFO`, `WARNING`, `ERROR` are internally stored as integers, even though the level parameter since version 3.2 accepts a string representation. You are right, it will most likely not change.

Another way would be to not import them but refer to them with their full name, e.g. `logging.DEBUG`

I don't know why I'm so reluctant for the use of strings for setting logging levels, I guess I don't like the look of the code. Anyway I will change it according to your change request. ;)

Reply:
Do what works for you! I just thought it looked cleaner without, since you can use strings. :smile: